tests/acceptance/features/cliDataExporter/import.feature

```diff
@@ -21,3 +21,32 @@ Feature: An administrator wants to import a user using the commandline
     And as "usertrash" file "AFolder/DeletedFile.txt" should exist
     And the content of file "AFolder/DeletedFolder/fileinfolder.txt" for user "usertrash" should be "text in file in deleted folder"
     And the content of file "AFolder/DeletedFile.txt" for user "usertrash" should be "text in deleted file"
+
+  Scenario: Restore and rename files in trashbin after import
+    Given a user has been imported from path "trashbinExport/usertrash" using the occ command
+    And the administrator has changed the password of user "usertrash" to "123456"
+    And user "usertrash" has restored the folder with original path "AFolder/DeletedFolder"
+    And user "usertrash" has restored the file with original path "AFolder/DeletedFile.txt"
+    And as "usertrash" file "AFolder/DeletedFolder/fileinfolder.txt" should exist
+    And as "usertrash" file "AFolder/DeletedFile.txt" should exist
+    When user "usertrash" moves file "AFolder/DeletedFile.txt" to "AFolder/RestoredDeletedFile.txt" using the WebDAV API
+    And user "usertrash" moves folder "AFolder/DeletedFolder" to "AFolder/RestoredDeletedFolder" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And as "usertrash" folder "AFolder/RestoredDeletedFolder" should exist
+    And as "usertrash" file "AFolder/RestoredDeletedFile.txt" should exist
+
+  Scenario Outline: Share files after import
+    Given a user has been imported from path "trashbinExport/usertrash" using the occ command
+    And the administrator has changed the password of user "usertrash" to "123456"
```

Comment:
Why changing password of user between import and sharing?? Is it making any difference?

Reply:
I don't know exactly why changing password is done in any scenarios for import. But I did it because in every early scenarios it was done so i thought it was mandatory.

Reply:
You can try running scenarios without those steps! Maybe you can find why.. :crossed_fingers: